Libraries/LibCore/SystemWindows.cpp

```diff
@@ -189,4 +189,11 @@ int getpid()
     return GetCurrentProcessId();
 }
 
+unsigned hardware_concurrency()
+{
+    SYSTEM_INFO si = {};
+    GetNativeSystemInfo(&si);
```

Comment:
This only gives the number of processors in a group. So up to 64. In my implementation I just left a comment here. This is also the limit for threading in msvc STL, so not sure if we even want to get the real number of processors. Also this could just be GetSystemInfo as we won't be running under WOW64.